.github/workflows/quality-tests.yml

```diff
@@ -0,0 +1,81 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
```

Comment:
We may also want to run on merges to `open-release/lilac.master`.

Reply:
Done.